android/libraries/rib-base/src/main/java/com/badoo/ribs/android/IntentCreator.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.ribs.android
+
+import android.content.Intent
+
+interface IntentCreator {
```

Comment:
We still have `com.uber.rib.core.IntentCreator` and `com.uber.rib.core.IntentCreatorImpl` in rib-android-test.
Should we update/remove them?

Reply:
Will do a whole cleanup run in the repo. Created #10 for this.